irctest/server_tests/chmodes/secret.py

```diff
@@ -0,0 +1,50 @@
+from irctest import cases
+from irctest.numerics import RPL_LIST
+
+
+class SecretChannelTestCase(cases.BaseServerTestCase):
+    @cases.mark_specifications("RFC1459")
+    def testSecretChannelListCommand(self):
+        """
+        <https://datatracker.ietf.org/doc/html/rfc1459#section-4.2.6>
+
+        "Likewise, secret channels are not listed
+        at all unless the client is a member of the channel in question."
+        """
```

Comment:
Please mark and quote Modern too.

https://modern.ircdocs.horse/#secret-channel-mode